blossom/templates/app/transcribing.html

```diff
@@ -3,58 +3,252 @@
+                    back so you can choose a new one.
+                </div>
+                <div class="d-grid gap-2 mx-2 mb-2">
+                    <a href="{% url 'app_unclaim' submission.id %}" class="btn btn-danger">Just want something
+                        different</a>
+                    <a href="{% url 'app_unclaim' submission.id %}?reason=rules" class="btn btn-outline-danger">
+                        Post breaks rules
+                    </a>
+                    <a href="{% url 'app_unclaim' submission.id %}?reason=removed" class="btn btn-outline-danger">
+                        Post was removed
+                    </a>
+                    <a href="{% url 'app_unclaim' submission.id %}?reason=op_transcribed"
+                       class="btn btn-outline-danger">
+                        OP already transcribed it
+                    </a>
+                </div>
+
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
                 </div>
             </div>
         </div>
-    {% endif %}
+    </div>
+
+    <div class="modal fade" id="rulesModal" tabindex="-1">
+        <div class="modal-dialog modal-lg modal-dialog-scrollable">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title">Rules for {{ submission.get_subreddit_name }}</h5>
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+                </div>
+                <div class="modal-body" id="rulesModalBody">
+                    {# This will be filled dynamically through the POWAH OF JAVASCWIPT #}
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+                </div>
+            </div>
+        </div>
+    </div>
+
 {% endblock %}
 
 {% block scripts %}
-    {% if source_url %}
-        <script src="{% static "js/openseadragon-bin-2.4.2/openseadragon.js" %}"></script>
-        <link rel="stylesheet" href="{% static "js/simplemde/simplemde.min.css" %}">
-        <script src="{% static "js/simplemde/simplemde.min.js" %}"></script>
-
-        <script type="text/javascript">
-            OpenSeadragon({
-                id: "imageViewer",
-                prefixUrl: "/static/js/openseadragon-bin-2.4.2/images/",
-                tileSources: {
-                    type: 'image',
-                    url: "{{ source_url }}",
-                    crossOriginPolicy: 'Anonymous',
-                    ajaxWithCredentials: false
+    <script src="{% static "js/openseadragon-bin-2.4.2/openseadragon.js" %}"></script>
+    <link rel="stylesheet" href="{% static "js/simplemde/simplemde.min.css" %}">
+    <script src="{% static "js/simplemde/simplemde.min.js" %}"></script>
+
+    <script type="text/javascript">
```

Comment:
For the scripts, I'd personally prefer them to be in a separate `.js` file and to just be imported here

Reply:
Hmm I see we have conditionals in there, so this might be harder than I thought

Reply:
yeah -- I try to put them in specific files when I can, but it's a lot easier to render the templates first rather than pass the variables through to external scripts.